src/main/java/nextstep/domain/Ladder.java

```diff
@@ -29,4 +28,9 @@ public List<String> getNames() {
     public List<String> getLines() {
         return lines.getLines();
     }
+
+    public LadderResult createLadderResult(List<String> prize) {
+        participants.move(lines);
```

Comment:
```suggestion
    private int getTarget(int position) {
        int target = position;
        for (Line line : lines) {
            System.out.println("current target : " + target);
            target = line.move(target);
        }
        return target;
    }
```

지금 코드는 participants을 중심으로 move()를 구현하며 lines를 전달하고 있다.
반대로 line.move()를 호출하면서 현재 위치 값을 전달하는 방식으로 구현하는 것은 어떨까?